llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -56,6 +56,8 @@ cl::opt<std::string> CASPath("cas", cl::desc("Path to CAS on disk."));
 cl::list<std::string> InputFiles(cl::Positional, cl::desc("Input object"));
 cl::opt<bool> ComputeStats("object-stats",
                            cl::desc("Compute and print out stats."));
+cl::opt<bool> ComputeStatsCSV("object-stats-csv",
```

Comment:
I would prefer this write to a file?

stdout is kind of crowded with root id and maybe other things.

Reply:
Maybe we can change `object-stats` to take a file argument; passing `-` will have it use `outs()` instead of opening a file (as usual); and then make this option `object-stats-format` that takes either `pretty` or `csv`.